serve.go

```diff
@@ -44,7 +45,16 @@ func Serve(c Connector) {
 func serve(c Connector) error {
 	err := initStandaloneModeLogger()
 	if err != nil {
-		return err
+		return fmt.Errorf("failed to initialize logger: %w", err)
+	}
+
+	target := os.Getenv("CONDUIT_STANDALONE_GRPC_TARGET")
+	if target == "" {
+		target = internal.DefaultStandaloneGRPCTarget
+	}
```

Comment:
I wouldn't set a default, if Conduit didn't set the env var it means it's an older version of Conduit and there are no connector utility services. The question is what we do in that case - I'd say throw an error with a descriptive error that points out the connector requires at least Conduit v0.11.0 (next release).

Reply:
@lovromazgon Good point on this being a way to detect older versions of Conduit. What I'll do is including a link to the 0.11.0 release which we could update later on with more information about this change.